dev/Pager/PrototypePager/PrototypePager.Events.cs

```diff
@@ -113,10 +114,12 @@ private void OnPagerDisplayModeChanged()
         {
             switch (this.PagerDisplayMode)
             {
+                case PagerDisplayModes.Auto:
+                    VisualStateManager.GoToState(this, NumberOfPages < 11 ? ComboBoxVisibleVisualState : NumberBoxVisibleVisualState, false);
```

Comment:
@MarissaMatt The proposal only says something about less than 11 and larger than 11, what about 11 itsself? Is case 11 resulting in NumberBox correct here? I think the spec might need some clarification for this.

Reply:
@chingucoding can you point me to where it talks about the "less than 11 and larger than 11" on the proposal? I can't seem to find any reference to it there.  I know the spec mentions it, but I'm interested to see what the proposal mentions.

Reply:
Sure, it is the first paragraph in this section: https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/savoyschuler/pagercontrol/active/PagerControl/PagerControl.md#creating-a-pager-control

It states:

"[...] NumberOfPages property is less than 11 and will show the number box mode if it is greater than 11."

Reply:
I updated the spec with that requirement but not the proposal so I will make sure they are both matching. The requirement should be if it's 10 pages or less then combo box will be chosen and if it's 10 or more then number box will be chosen.

Reply:
I'm still a little confused..

So should it be:
`if NumberOfPages <= 10 then use ComboBox else use NumberBox`
or
`if NumberOfPages < 10 then use ComboBox else use NumberBox`

Reply:
It should be if NumberOfPages < 10 then use ComboBox else use NumberBox

Reply:
Great thanks! I'll update the prototype